engine/src/main/java/org/entando/entando/aps/util/UrlUtils.java

```diff
@@ -218,11 +219,15 @@ public EntUrlBuilder paths(String ... u){
             return this;
         }
 
-        public URI build(){
-            UriBuilder builder = UriBuilder.fromUri(url);
+        public URI build() {
+            UriBuilder builder = UriComponentsBuilder.fromHttpUrl(url);
```

Comment:
has this method the same behavior? (from uri from url sound strange to me ...)

Reply:
I used `fromUriString`, but we are passing an URL in any case.